### PR TITLE
Don't create a CovidKey object if there was a failure

### PR DESCRIPTION
**Pull request by PierrePaul**
Moving the covidkey logic around to prevent creating an obj when there is an error

https://trello.com/c/mlZ4q1X1/230-dont-create-a-key-object-unless-the-key-retrieval-worked

### CODE REVIEW

**Comment by PierrePaul** [comment]
Screenshot_2020-08-04 Give patient this key — COVID Alert Portal(1)

File: covid_key/views.py [hunk 1]
```diff
@@ -31,31 +33,54 @@ def code(request):
 def _generate_key(request):
     token = settings.API_AUTHORIZATION
     diagnosis_code = "0000000000"
+    covid_key = None
     if token:
+        print(token)
```
Comment (pcraig3) [inline covid_key/views.py:38]: Let's remove this.

File: covid_key/views.py [hunk 2]
```diff
@@ -31,31 +33,54 @@ def code(request):
 def _generate_key(request):
     token = settings.API_AUTHORIZATION
     diagnosis_code = "0000000000"
```
Comment (pcraig3) [inline covid_key/views.py:35]: We can also default this to `""`, since we don't want to display the zeroes.

Comment (PierrePaul) [reply covid_key/views.py:35]: In development Im still showing the 000 000 0000.
As soon as token and api_endpoint is provided, the error will be shown instead.

Comment (pcraig3) [reply covid_key/views.py:35]: gotcha, sounds good.

**Pull request merged**